### PR TITLE
gh-418: add variable names for `Yields` vars

### LINKED ISSUE #418: Single-line yields render as parameter names in docs
Fantastic effort, particularly with the many changes in specs! There is a remaining issue with single-line yields but it's so minuscule that I want to tackle it separately. Towards a new release!

_Originally posted by @ntessore in https://github.com/glass-dev/glass/pull/381#pullrequestreview-2433892737_

### PR DESCRIPTION
**Pull request by paddyroddy**
Closes #418. Without these names, they render as "special-cased".

### CODE REVIEW

File: glass/points.py [hunk 1]
```diff
@@ -342,11 +342,11 @@ def uniform_positions(
...
     Yields
     ------
-    lon:
+    lon
         Columns of longitudes for the sampled points.
-    lat:
+    lat
         Columns of latitudes for the sampled points.
-    count:
+    count
```
Comment (paddyroddy) [inline glass/points.py:349]: They shouldn't be here. Unintentionally introduced in #381.

Comment (ntessore) [reply glass/points.py:349]: Are these not necessary to make the names show up as names (bold) instead of types (italic)?

Comment (ntessore) [reply glass/points.py:349]: Excellent, it renders fine on RTD! 🚀

**Review by ntessore — APPROVED** [review]
Thanks @paddyroddy!

File: glass/points.py (hunk 1 above)
Comment (paddyroddy) [reply glass/points.py:349]: Nope, I think it expects the type if you have them. We don't use colons anywhere else (like in the parameters).

**Pull request merged**